src/mbr.rs

```diff
@@ -166,6 +166,20 @@ impl ProtectiveMBR {
         self
     }
 
+    /// Returns the given partition (0..=3). Panics if the partition index is invalid.
```

Comment:
Should this panic though? Someone using this library will not be able to recover from this error.

Reply:
That's fair! The getter/setter have now been changed to return an `Option` and return `None` if the index was invalid.